packages/forms/src/hooks/useCommonInputStyles.ts

```diff
@@ -1,16 +1,29 @@
 import { StyleRecord } from "../types";
 
-type UseCommonInputStyles = () => StyleRecord;
+type UseCommonInputStyles = (type?: string) => StyleRecord;
 
-const useCommonInputStyles: UseCommonInputStyles = () => {
-  return {
+const useCommonInputStyles: UseCommonInputStyles = (type) => {
+  const defaults = {
     "data-h2-border-style": "base(solid)",
     "data-h2-border-width": "base(1px)",
     "data-h2-outline": "base(none) base:focus-visible(2px solid focus)",
     "data-h2-outline-offset": "base(2px)",
-    "data-h2-padding": "base(x.5)",
     "data-h2-radius": "base(rounded)",
+    "data-h2-color": "base(black)",
+    "data-h2-padding": "base(x.5)",
   };
+  let selectIcon = {};
+  if (type === "select") {
```

Comment:
Open to that for sure. I just generally find it more helpful to keep everything together for context when I'm working with styles. It's helpful to see how the common styles might influence the conditional styles in situ IMO.

Reply:
My preference would be to split these up.  Usually we try to compose smaller more-focused functions together instead of larger functions that do more than one thing.

That said, if you'd prefer to keep them together I suggest a little refactoring to help the function make more sense:
- rename function to `useInputStyles`
- rename parameter to `inputType`, or maybe `stylesType`?
- constrain the the parameter type to a union of just the values the function can handle instead of `string`
    - just "common" and "default" right now, I guess, with a default value of "default"

Reply:
Awesome, thanks @petertgiles! I've gone ahead with your suggestions in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8601/commits/50f0f7030f9f74f0179aeda41855f28d21d4a97a and https://github.com/GCTC-NTGC/gc-digital-talent/pull/8601/commits/f851ee01d20e47a6cae12df0a8632d458038e672

Reply:
Nice, turned out great!